src/main/scala/thurloe/service/ThurloeServiceActor.scala

```diff
@@ -25,8 +25,8 @@ class ThurloeServiceActor(config: Config) extends Actor with FireCloudProtectedS
 
   override def receive = runRoute(
       swaggerUiResourceRoute ~
-      statusRoute ~
-      fireCloudProtectedRoutes
+      fireCloudProtectedRoutes ~
+      statusRoute
```

Comment:
@aednichols - Can you test this without the swap? If it still works in the original, please revert.

Reply:
Ignore this, sorry.